introduction.md

```diff
@@ -2,3 +2,5 @@ This is some text. My favorite Code School path is HTML & CSS.
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?